src/styles/partials/_variables.scss

```diff
@@ -1,13 +1,15 @@
 // FONT FACES
-$font-family-serif: Baskerville, 'Baskerville Old Face', 'Hoefler Text', Georgia, Garamond, 'Times New Roman', serif;
-$font-family-sans-serif: "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
+$font-family-serif: Baskerville, "Baskerville Old Face", "Hoefler Text", Georgia,
```

Comment:
Do you want to change these to the new brand font? I think the screenshots look a little odd because it's using the new brand typeface in the logo, but the rest of the site is still using the old one.

Reply:
Good point. But the new sans-serif font has actually not been chosen just yet :) (will be different, but fitting with the logo)